react-native/src/blueDarkTheme.ts

```diff
@@ -0,0 +1,31 @@
+/**
+Copyright (c) 2020-present, Eaton
+
+All rights reserved.
+
+This code is licensed under the BSD-3 license found in the LICENSE file in the root directory of this source tree and at https://opensource.org/licenses/BSD-3-Clause.
+**/
+
+import { configureFonts, DefaultTheme } from 'react-native-paper';
+import { blue, red, darkBlack, black, lightBlue, gray } from '@pxblue/colors';
+import { fontConfig } from './shared';
+import { Theme } from 'react-native-paper/lib/typescript/src/types';
+
+export const blueDarkTheme: Theme = {
+    ...DefaultTheme,
+    dark: true,
+    roundness: 4,
+    fonts: configureFonts(fontConfig),
+    colors: {
+        ...DefaultTheme.colors,
+        primary: blue[500],
+        accent: lightBlue[500],
+        background: darkBlack[100],
+        surface: black[900],
+        error: red[500],
+        text: gray[300],
```

Comment:
black 50

Reply:
We have two colors here, one is black 50 for primary text, one is black 200 for secondary text.